Languages/es.xml

```diff
@@ -119,7 +119,7 @@ Thanks: Jhovany200, Oleada, nomemolesteis@gmail.com for translation
     <String Target="基础支撑库缺失，引导修复相关功能无法使用。" Link="Falta soporte fundamental de la biblioteca. No se pueden usar las funciones relacionadas con boot-fix."/>
     <String Target="Dism++需要管理员权限才能运行，请右键 以管理员身份运行 然后继续。" Link="Dism++ necesita derechos administrativos para ejecutarse. Clic botón derecho y seleccione Ejecutar como administrador para ejecutarlo."/>
 
-    <String Target="你需要增量保存还是直接保存？" Link="¿Desea guardar de forma directa o incremental?"/>
+    <String Target="你需要增量保存还是直接保存？" Link="¿Guardar de forma directa o en incremental?"/>

```

Comment:
Translation incorrect. No exist word incremental in spanish
¿Guardar de forma directa o en incrementos?